plugin/trino-redshift/src/test/java/io/trino/plugin/redshift/TestRedshiftTypeMapping.java

```diff
@@ -658,8 +659,9 @@ public void testTimestampWithTimeZoneOverflow()
 
         // The max timestamp with time zone value in Redshift is larger than Trino
         try (TestTable table = new TestTable(getRedshiftExecutor(), TEST_SCHEMA + ".timestamp_tz_max", "(ts timestamptz)", ImmutableList.of("TIMESTAMP '294276-12-31 23:59:59' AT TIME ZONE 'UTC'"))) {
-            assertThat(query("SELECT * FROM " + table.getName()))
-                    .failure().hasMessage("Millis overflow: 9224318015999000");
+            assertThatThrownBy(() -> computeActual("SELECT * FROM " + table.getName()))
```

Comment:
Why not replace `failure` with `nonTrinoExceptionFailure`? Using `assertThatThrownBy` hides a problem that we should change the exception.